extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -86,6 +87,28 @@ export class AmpTiktok extends AMP.BaseElement {
     );
   }
 
+  /** @override @nocollapse */
+  static createLoaderLogoCallback(element) {
+    const html = htmlFor(element);
+    return {
+      color: '#FFFFFF',
+      content: html`<svg
+        width="38"
+        height="38"
+        viewBox="0 0 72 72"
+        fill="none"
+        style="margin: 17px;"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M22.96.04C25.35 0 27.72.02 30.09 0c.14 2.8 1.14 5.67 3.18 7.65 2.04 2.03 4.93 2.96 7.73 3.28v7.38a19.26 19.26 0 01-10.6-3.48c-.02 5.36.01 10.7-.04 16.04a14.01 14.01 0 01-2.47 7.23 13.55 13.55 0 01-10.77 5.88A13.2 13.2 0 019.7 42.1a13.82 13.82 0 01-6.65-10.47c-.04-.92-.06-1.84-.02-2.73a13.77 13.77 0 014.7-9.1 13.5 13.5 0 0111.21-3.16c.04 2.72-.07 5.43-.07 8.15a6.32 6.32 0 00-5.5.68 6.35 6.35 0 00-2.49 3.2c-.38.94-.27 1.97-.25 2.96.44 3 3.31 5.53 6.38 5.26a6.14 6.14 0 005.05-2.95c.34-.6.73-1.23.75-1.95.18-3.28.1-6.54.13-9.82.01-7.4-.02-14.76.03-22.13z"
+          fill="#fff"
+          style="transform: translate(14px, 13px);"
+        />
+      </svg>`,
```

Comment:
Suggested a few modifications in the following:
- Remove width/height on `svg` as the element is sized via loader logic to be `72x72`
- Remove `fill` which is not an attribute on `svg`
- Remove `style` on `svg` and `path` and instead prefer a `d` value which is centered within the `viewBox` dimensions. -- Not sure where the original value came from / if there are more formal ways to create it centered within certain dimensions, but I used https://yqnn.github.io/svg-path-editor/ to get the suggested value. 
```suggestion
      content: html`<svg viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
        <path
          d="M 36.1926 24.0392 C 37.4471 24.0182 38.691 24.0287 39.935 24.0182 c 0.0735 1.4697 0.5984 2.9761 1.6691 4.0153 c 1.0708 1.0655 2.5877 1.5536 4.0573 1.7216 v 3.8736 a 10.1092 10.1092 90 0 1 -5.5637 -1.8266 c -0.0105 2.8134 0.0052 5.6162 -0.021 8.4191 a 7.3536 7.3536 90 0 1 -1.2965 3.7949 a 7.1121 7.1121 90 0 1 -5.653 3.0863 A 6.9284 6.9284 90 0 1 29.2327 46.1157 a 7.2538 7.2538 90 0 1 -3.4905 -5.4955 c -0.021 -0.4829 -0.0315 -0.9658 -0.0105 -1.4329 a 7.2276 7.2276 90 0 1 2.4669 -4.7764 a 7.0859 7.0859 90 0 1 5.8839 -1.6586 c 0.021 1.4277 -0.0367 2.8501 -0.0367 4.2778 a 3.3172 3.3172 90 0 0 -2.8868 0.3569 a 3.333 3.333 90 0 0 -1.307 1.6796 c -0.1995 0.4934 -0.1417 1.034 -0.1312 1.5536 c 0.2309 1.5746 1.7374 2.9026 3.3487 2.7609 a 3.2228 3.2228 90 0 0 2.6506 -1.5484 c 0.1785 -0.3149 0.3832 -0.6456 0.3937 -1.0235 c 0.0945 -1.7216 0.0525 -3.4327 0.0682 -5.1543 c 0.0052 -3.8841 -0.0105 -7.7472 0.0157 -11.6156 z"
          fill="#fff"
        />
      </svg>`
```

Reply:
In this case I was using width and height to slightly shrink the svg path so that it would fit inside the loading circle properly - would it be better to do this by modifying the svg path itself as well?

Reply:
I personally prefer modifying path over combining a specific path with css, a bit cleaner IMO

Reply:
@kristoferbaxter I have requested a new SVG which is centered inside the viewbox, but In the interest of landing this PR I'd like to ship it as is, and will update to the new path once the new svg's come back.

Reply:
@rebeccanthomas It's okay to submit is as-is as long as we describe the issue with the SVG content as a justification for the transforms/attributes. I agree with @caroqliu that it's cleaner to modify the path and viewbox itself, but it's ok to submit as-is.

Reply:
Let's please also add a comment about the second svg that's rendered conditionally.